04_numeronym/vinilapeddireddy/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(s ...string) []string {
+	var numeronyms []string
```

Comment:
This is fine, but if you really care about runtime you could optimise by preallocating the size of `numeronyms`:
```go
numeronyms := make([]string, len(vals))
for i, val := range vals {
		numeronyms[i] =  numeronym(val)
}
```

Reply:
Done